judge/fixtures/language_all.json

```diff
@@ -942,5 +942,21 @@
         "description": "",
         "extension": "cpp"
     }
+},
+{
+    "model": "judge.language",
+    "pk": 77,
+    "fields": {
+        "key": "NODEJS",
+        "name": "JavaScript (Node.js)",
+        "short_name": "Node.js",
```

Comment:
The short name is used on the submission listing. For consistency, it should be all uppercase. If you leave it as `null`, then `key` will be used instead (which seems fine in this case).